ERCS/erc-7913.md

```diff
@@ -78,9 +64,19 @@ Consistency with existing systems (ecrecover and ERC-1271) requires the message
 
 ## Backwards Compatibility
 
-This ERC proposed a new contract interface, with no consequences to the existing system.
+A system can support ERC-7913 signatures alongside EOAs and ERC-1271 in the following way.
```

Comment:
```suggestion
A system can support [ERC-7913](./eip-7913.md) signatures alongside EOAs and ERC-1271 in the following way.
```

Reply:
Accepted without the link since a link to itself seems odd.

Reply:
If we write ERC-7913, then we need the link !

There is a "rule" for eip document enforces that the first reference to an ERC/EIP must be the link. That rule also apply to self. If you don't want the link you need to rephrase things.